cli/version_check_test.go

```diff
@@ -12,11 +12,6 @@ func TestCheckTerraformVersionMeetsConstraintEqual(t *testing.T) {
 	testCheckTerraformVersionMeetsConstraint(t, "v0.9.3", ">= v0.9.3", true)
 }
 
-func TestCheckTerraformVersionMeetsConstraintGreaterDev(t *testing.T) {
-	t.Parallel()
-	testCheckTerraformVersionMeetsConstraint(t, "v0.9.4-dev", ">= v0.9.3", true)
-}
```

Comment:
Curious why this was removed?

Reply:
Oh, it's in the commit message, but not obvious from the PR: the `go-version` library we use under the hood was updated as part of this PR, and they changed how they handle version comparisons between "pre-releases" (e.g., v0.9.4-dev), so this comparison isn't actually valid anymore.

Reply:
Ah got it. Thanks for the explanation! I should have scanned the commit messages.